src/test/java/uk/gov/justice/hmpps/casenotes/services/CaseNoteServiceTest.java

```diff
@@ -155,6 +159,89 @@ public void getCaseNote_callElite2() {
         assertThat(caseNote.getEventId()).isEqualTo(12345);
     }
 
+    @Test
```

Comment:
bit naff to add more tests to a java file - would be better to either rewrite the file in kotlin or create a new kotlin file with the tests in